render/render.js

```diff
@@ -747,7 +747,7 @@ module.exports = function($window) {
 		}
 	}
 	function isFormAttribute(vnode, attr) {
-		return attr === "value" || attr === "checked" || attr === "selectedIndex" || attr === "selected" && vnode.dom === $doc.activeElement || vnode.tag === "option" && vnode.dom.parentNode === $doc.activeElement
+		return attr === "value" || attr === "checked" || attr === "selectedIndex" || attr === "selected" || vnode.tag === "option" && vnode.dom.parentNode === $doc.activeElement
```

Comment:
Could you please revert this for now and only fix the one issue with `input.value`? The comment in the bug regarding this was me noting I wanted to *investigate* (since it appeared similar in use), not that I actively wanted to *change* it.

Reply:
Will do!